tasklist/client/src/Tasks/Task/Variables/LoadingTextarea/index.tsx

```diff
@@ -49,7 +49,7 @@ const LoadingTextarea: React.FC<Props> = ({
         data-testid="textarea-loading-overlay"
       >
         <div className={styles.overlay}>
-          <Loading className={styles.loading} withOverlay={false} />
```

Comment:
Not sure why the TS plugin is broken when running from the CLI, it works from the IDE